frontend/lib/tests/pages/letter-request.test.tsx

```diff
@@ -33,5 +27,27 @@ describe('landlord details page', () => {
     const { mock } = pal.appContext.updateSession;
     expect(mock.calls).toHaveLength(1);
     expect(mock.calls[0][0]).toEqual({ letterRequest: { updatedAt, mailChoice } });
+  }
+
+  it('works when user chooses to mail the letter themselves', async () => {
+    const pal = new AppTesterPal(<LetterOfComplaintRoutes />, {
+      url: Routes.loc.preview,
+      session: { letterRequest: PRE_EXISTING_LETTER_REQUEST }
+    });
+    clickButtonAndExpectChoice(pal, /mail this myself/i, LetterRequestMailChoice.USER_WILL_MAIL);
+  });
+
+  it('works when user wants us to mail the letter', async () => {
+    const pal = new AppTesterPal(<LetterOfComplaintRoutes />, {
+      url: Routes.loc.preview,
+      session: { letterRequest: PRE_EXISTING_LETTER_REQUEST }
+    });
+    pal.clickButtonOrLink(/ready to send/i);
+    await pal.rt.waitForElement(() => pal.getDialogWithLabel(/ready to go/i));
+
+    // This is a workaround for https://github.com/davidtheclark/focus-trap-react/issues/24.
+    await pause(10);
```

Comment:
Just documented the rationale for this workaround at https://github.com/davidtheclark/focus-trap-react/issues/24#issuecomment-431424268.